src/main/java/seedu/address/logic/commands/DeleteCustomerCommand.java

```diff
@@ -36,6 +38,8 @@ public DeleteCustomerCommand(Phone targetPhone) {
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Customer> lastShownList = model.getFilteredCustomerList();
```

Comment:
Currently, there is one issue with this. Steps to replicate:

1. Use find command to Find Customer B
2. Delete Customer A
3. The customer phone number provided is invalid.

I would suggest to use the `getCustomerWithPhone` to find the customer instead, `filteredCustomerList` is a bit unreliable